packages/strapi-plugin-settings-manager/admin/src/containers/Home/index.js

```diff
@@ -179,18 +243,95 @@ export class Home extends React.Component { // eslint-disable-line react/prefer-
 
   renderListButtonLabel = () => `list.${this.props.params.slug}.button.label`;
 
+  renderPopUpFormLanguage = (section, props) => (
+    map(section.items, (item, key) => (
+      <div key={key}>
+        <form>
+          {props.renderInput(item, key)}
+        </form>
+      </div>
+    ))
+  )
+
+  renderRowDatabase = (props, key, listStyles) => {
+    let provider;
+
+    // TODO provider
+    switch(props.provider) {
```

Comment:
Not a big fan of this portion. The switch is too much static. What happens if we support another type of databases in the future? Moreover, it should be based on the client property and returned by the back-end.

For example, if the `connector` is `strapi-bookshelf` and the database is `postgres`, we should display `PG`. If the `connector`is `strapi-mongoose` it should display `M`. If the `connector `strapi-redis`, we should display `R`.

The rules are simples:

1. Take the first letter of the database.
2. Except for the SQL databases (PG => postgres, MS => mysql, L => sqllite)